src/dodal/devices/athena/panda/flyingpanda.py

```diff
@@ -0,0 +1,44 @@
+from typing import Dict, Iterator
+
+from bluesky.protocols import Descriptor, Flyable, PartialEvent
+from ophyd_async.core import AsyncStatus, wait_for_value
+from ophyd_async.panda import PandA
+
+import dodal.devices.athena.panda.tables as tables
+
+
+class FlyingPanda(Flyable):
+    def __init__(self, panda):
+        self.dev: PandA = panda
+        self._frames = []
+
+    @property
+    def name(self) -> str:
+        return self.dev.name
+
+    async def set_frames(self, frames):
+        table = tables.build_table(*zip(*frames))
+        await self.dev.seq[1].tables.set(table)
+
+    @AsyncStatus.wrap
+    async def kickoff(self) -> None:
+        # NOTE: the first of these DOES NOT WORK. This is a bug and is being tracked:
+        # https://github.com/bluesky/ophyd-async/issues/13
+        # you will need to actually do caput/pvput to make prescale_units ms.
+        await self.dev.seq[1].prescale_units.set("ms")
+        await self.dev.seq[1].prescale.set(1)
+        await self.dev.seq[1].enable.set("ZERO")
+        await self.dev.seq[1].repeats.set(1)
+        await self.dev.seq[1].enable.set("ONE")
```

Comment:
Could: `ms`, `ZERO` and `ONE` should all be pulled into constants, probably the place for them is in `ophyd-async`

Reply:
We've discussed and think FlyingPanda should be in ophyd-async alongside Panda as you suggest, so will resolve this and add change there.